content/docs/command-reference/remote/list.md

```diff
@@ -1,12 +1,6 @@
 # remote list
 
-List all available data remotes.
```

Comment:
@jorgeorpinel no related to this PR? Please let's try to do this in Regular updates or separate PRs

Reply:
Yeah sorry I decided to just standardize how all the commands with indices link to the subcommands the same way its done in metrics. I've been sloppy with PR separation lately because of all the battlefronts but will be more careful.